pyproject.toml

```diff
@@ -71,6 +71,12 @@ include = ["shapely", "shapely.*"]
 [tool.setuptools.package-data]
 "shapely" = ["*.pxd"]
 
+[tool.cibuildwheel]
+skip = ["pp*", "*_i686", "*_ppc64le", "*_s390x"]
```

Comment:
Not too important, but just wondering to what extent it is actually needed to list eg `"*_i686` given we are running with explicit many/musllinux docker images (I would not expect that it tries to build such wheels by default in a manylinux2014_x86-64 image)

Reply:
I think it makes it more explicit to not build these. It changes the output from (e.g.) a command `cibuildwheel --print-build-identifiers`